docs/configuration.md

```diff
@@ -37,7 +37,10 @@ Before you can use `gphotos-uploader-cli`, you must enable the Photos Library AP
 1. Make sure you're logged in into the Google Account where your photos should be uploaded to.
 1. Start by [creating a new project](https://console.cloud.google.com/projectcreate) in Google Cloud Platform and give it a name (example: _Google Photos Uploader_).
 1. Enable the [Google Photos Library API](https://console.cloud.google.com/apis/library/photoslibrary.googleapis.com) by clicking the <kbd>ENABLE</kbd> button.
-1. Configure the [OAuth consent screen](https://console.cloud.google.com/apis/credentials/consent) by setting the application name (example: _gphotos-uploader-cli_) and then click the <kbd>Save</kbd> button on the bottom.
+1. Configure the [OAuth consent screen](https://console.cloud.google.com/apis/credentials/consent) by
+   - setting the application name (example: _gphotos-uploader-cli_)
+   - click <kbd>+ ADD USERS</kbd> under "Test users", and add your email you will authenticate with
```

Comment:
Uhm... This has not been needed to me. I want to understand better when this step is needed and when it's not 😓

Reply:
I'm not sure why it would be different for us, i had assumed you had already enabled your app so you didn't think to put it in the docs. I also have [Advanced Protection](https://landing.google.com/advancedprotection/) enabled on my google account, perhaps that requires the extra validation. I got auth errors when I tried using the CLI without this step.

No pressure to add this to the docs if you don't want, but I wanted to contribute to make the docs match the required steps (at least for a subset of users apparently). Also, thanks for sharing the project, it was super helpful for me. It let me upload 94 albums overnight from a scanned photo archive project. It would have taken me many hours of clicking and waiting otherwise.

Reply:
I set up gphotos-uploader-cli today and I too had to add myself as a test user.

Going back to step 4:
> Configure the [OAuth consent screen](https://console.cloud.google.com/apis/credentials/consent) by setting the application name (example: gphotos-uploader-cli) and then click the Save button on the bottom.

Here for me at least, I am only able to select a `User Type` of `External`. If I try to select `Internal` it says:

>  Because you’re not a Google Workspace user, you can only make your app available to external (general audience) users. 

And the description of `External` says:

>  Available to any test user with a Google Account. Your app will start in testing mode and will only be available to users you add to the list of test users. Once your app is ready to push to production, you may need to verify your app.

So for those of us who are not google workspace users (whatever that is), we get an app in `testing mode` which means we have to add ourselves to the list of authorized users for the app.


I suppose then this MR could be improved by having the instructions mention this extra step will be needed for if the `User Type` is `External`.